documentation/modules/collected-logs-cr-info.adoc

```diff
@@ -19,24 +19,19 @@ The `must-gather` tool collects the following logs and CR files in an archive fi
 ** `Plan` CR: Defines the VMs and storage and network mapping.
 
 * Logs:
-** `importer` pod: Disk-to-data-volume conversion log.
-+
-The pod naming convention is `importer-<migration_plan>-<vm_id>-<5_char_id>` for VMware and `importer-<migration_plan>-<partial_vm_id><5_char_id>` for {rhv-short}, for example, `importer-mig-plan-ed90dfc6-9a17-4a8btnfh`, where `ed90dfc6-9a17-4a8` is the partial VM ID and `btnfh` is the 5-character ID.
-
-** `conversion` pod: VM conversion log. The `conversion` pod runs `virt-v2v`, which installs and configures device drivers on the PVCs of the VM.
-+
-The pod naming convention is `<migration_plan>-<vm_id>-<5_char_id>` for VMware and `<migration_plan>-<partial_vm_id><5_char_id>` for {rhv-short}.
+** `importer` pod: Disk-to-data-volume conversion log. The `importer` pod naming convention is `importer-<migration_plan>-<vm_id><5_char_id>`, for example, `importer-mig-plan-ed90dfc6-9a17-4a8btnfh`, where `ed90dfc6-9a17-4a8` is a truncated {rhv-short} VM ID and `btnfh` is the generated 5-character ID.
+** `conversion` pod: VM conversion log. The `conversion` pod runs `virt-v2v`, which installs and configures device drivers on the PVCs of the VM. The `conversion` pod naming convention is `<migration_plan>-<vm_id><5_char_id>`.
 
 ** `virt-launcher` pod: VM launcher log. When a migrated VM is powered on, the `virt-launcher` pod runs `QEMU-KVM` with the PVCs attached as VM disks.
-** `forklift-controller` pod: Filtered for the migration plan, virtual machine, or namespace.
-** `forklift-must-gather-api` pod: Filtered for the migration plan, virtual machine, or namespace.
+** `forklift-controller` pod: Filtered for a migration plan, virtual machine, or namespace target.
+** `forklift-must-gather-api` pod: Filtered for a migration plan, virtual machine, or namespace target.
 +
```

Comment:
target namespace sounds better (IMO)

Reply:
"target namespace" probably sounds better because we use it a lot for OpenShift migration but does not have the same meaning here. 

This is a must-gather collection for a target (plan/VM/namespace). Perhaps "... namespace must-gather target" would work. I will think about how to rewrite this.